README.md

```diff
@@ -16,16 +16,30 @@ Add a `.env` file in the root of the project. You will need to add config for co
 
 Just run `yarn` in the root of the project.
 
-### Setting up the database
-
-Run `yarn db:init` to create the database, create the tables and insert some initial data.
-
 ### Start the app
 
+Locally:
+
 ```sh
 yarn start
 ```
 
+Docker:
+
+```sh
+docker-compose up -d && yarn start:app
+```
+
 ### Scripts
 
 For a list of all scripts see the `scripts` field in the [package.json](package.json).
+
+### DB
```

Comment:
Can we use `Database` instead of `DB`, same with the body of this section

Reply:
👍